GOVERNANCE.md

```diff
@@ -0,0 +1,44 @@
+NOTE: The following is a snapshot of a perpetually changing state of affairs in
+the WGPU community. It is not a binding contract between users and
+decision-makers of the WGPU project.
+
+---
+
+The **WGPU project** is an ecosystem that _enables application authors to write
+portable and performant graphics programs_. It was originally conceived to
+provide an implementation of WebGPU for Firefox as the standard evolved and
+settled into something that could be shipped on all web browsers. WGPU has also
+enjoyed much contribution and use from other projects that requires graphics
+programming. We expect that these sorts of users will continue for the lifetime
+of project, and we embrace these contributors' needs and effort as the lifeblood
+of WGPU.
+
+## Mission
+
+The WGPU community seeks to realize the following directives through the
+project: it…
+
+1. …provides an implementation of the WebGPU API that is…
+    1. …correct and fully conformant.
+    1. …portable across all major platforms.
+    1. …performant enough to enable demanding applications.
+1. …serves as a platform of experimentation for:
+    1. …WebGPU standards development.
+    1. …native application authors.
+
+## Decision-making
+
+The WGPU community's decision-making is influenced by the following
+groups:
+
+* Connor Fitzgerald (@cwfitzgerald)
```

Comment:
Let's call this community leadership and have me, wumpf, groves

Reply:
Implemented with fe1d09d0a.